setup/steps/autoexplain_recommended_settings.go

```diff
@@ -0,0 +1,519 @@
+package steps
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	survey "github.com/AlecAivazis/survey/v2"
+	"github.com/guregu/null"
+	"github.com/lib/pq"
+
+	s "github.com/pganalyze/collector/setup/state"
+	"github.com/pganalyze/collector/setup/util"
+)
+
+// N.B.: this needs to happen *after* the Postgres restart so that ALTER SYSTEM
+// recognizes these as valid configuration settings
+var ConfigureAutoExplain = &s.Step{
+	Kind:        s.AutomatedExplainStep,
+	Description: "Review auto_explain settings",
+	Check: func(state *s.SetupState) (bool, error) {
+		if state.DidAutoExplainRecommendedSettings ||
+			(state.Inputs.SkipAutoExplainRecommended.Valid && state.Inputs.SkipAutoExplainRecommended.Bool) {
+			return true, nil
+		}
+		logExplain, err := util.UsingLogExplain(state.CurrentSection)
+		if err != nil || logExplain {
+			return logExplain, err
+		}
+
+		return false, nil
+	},
+	Run: func(state *s.SetupState) error {
+		var doReview bool
+		if state.Inputs.Scripted {
+			if state.Inputs.SkipAutoExplainRecommended.Valid {
+				doReview = !state.Inputs.SkipAutoExplainRecommended.Bool
+			}
+		} else {
+			err := survey.AskOne(&survey.Confirm{
+				Message: "Review auto_explain configuration settings?",
+				Default: false,
+				Help:    "Optional, but will ensure best balance of monitoring visibility and performance; review these settings at https://www.postgresql.org/docs/current/auto-explain.html#id-1.11.7.13.5",
```

Comment:
Probably better to link to https://pganalyze.com/docs/explain/setup/auto_explain instead - that also makes it easier to update in the future.